.github/workflows/integration-tests-on-release.yaml

```
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions: ["v1.24.17", "v1.25.16", "v1.27.10", "v1.29.1"]
+        kubernetes-versions: ["v1.25.16", "v1.25.16", "v1.27.10", "v1.29.1"]
```

```suggestion
        kubernetes-versions: ["v1.25.16", "v1.27.x", "v1.28.x", "v1.29.1"]
```

similar changes below

---

Updated